developer/wip/for-adopters/Getting Started.md

```diff
@@ -0,0 +1,123 @@
+technologies and integrated with the EDC Control Plane by implementing
+the [Data Plane Signaling API](https://github.com/eclipse-edc/Connector/blob/main/docs/developer/data-plane-signaling/data-plane-signaling.md).
+
+### Federated Catalog
+
+The Federated Catalog (FC) is responsible for crawling and caching data catalogs from other participants. The FC builds
+a local cache that can be queried or processed without resorting to complex distributed queries across multiple
+participants.
+
+### Identity Hub
+
+The Identity Hub securely stores and manages W3C Verifiable Credentials, including the presentation of VCs and the
+issuance and re-issuance process.
+
+### The Big Picture: The Dataspace Context
+
+EDC components are deployed to create a dataspace ecosystem. It is important to understand that there is no such thing
+as "dataspace software." At its most basic level, a dataspace is simply a context between two participants:
+
+![[Big Picture.svg]]
+
+The **Federated Catalog** fetches data catalogs from other participants. A **Connector** negotiates a contract agreement
+for data access between two participants and manages data transfers using a data plane technology. The **Identity Hub**
+presents verifiable credentials that a participant connector uses to determine whether it trusts and should grant data
+access to a counterparty.
+
+The above EDC components can be deployed in a single runtime process (e.g., K8S ReplicaSet) or a distributed topology (
+multiple ReplicaSets or clusters). The connector components can be further decomposed. For example, multiple control
+plane components can be deployed within an organization in a federated manner where departments or subdivisions manage
+specific instances termed `Management Domains`.
+
+### Customizing the EDC
+
+EDC was designed with the philosophy that one size does not fit all. Before deploying an EDC-powered data sharing
+ecosystem, you'll need to build customizations and bundle them into one or more distributions. Specifically:
+
+- **Policies** - Create a set of policies for data access and usage control. EDC adopts a code-first approach, which
+  involves writing *policy functions*.
+- **Verifiable Credentials** - Define a set of W3C Verifiable Credentials for your use cases that your policy functions
+  can process. For example, a credential that identifies a particular partner type.
+- **Data transfer types** - Define a set of data transfer technologies or types that must be supported. For example,
+  choose out-of-the-box support for HTTP, S3-based transfers, or Kafka. Alternatively, you can select your preferred
+  wire protocol and implement a custom data plane.
+- **Backend connectivity** - You may need to integrate EDC components with back-office systems. This is done by writing
+  custom extensions.
+
+Third parties and other open source projects distribute EDC extensions that can be included in a distribution. These
```

Comment:
I'm assuming at some point we'll have a definition of terms, and when we do, we should add a link from "distribution" to it
```suggestion
Third parties and other open source projects distribute EDC extensions that can be included in a [distribution](). These
```

Reply:
Yes, we should have a glossary once we get done.